x-pack/plugins/security_solution/public/resolver/view/panels/node_events_of_type.test.tsx

```diff
@@ -15,7 +15,7 @@ import { urlSearch } from '../../test_utilities/url_search';
 const resolverComponentInstanceID = 'resolverComponentInstanceID';
 
 // FLAKY: https://github.com/elastic/kibana/issues/85714
```

Comment:
Let's remove this line :)

Reply:
Done :) And tested again at `stress -c 256` and looks good.